CONTRIBUTING.md

```diff
@@ -102,14 +107,24 @@ pytest -m not slow
 ...
 ```
 
+### Running all the tests
+
+**Important**: If you want to run **all** the tests locally, you'll need **all** document stores running in the background
```

Comment:
I would suggest to change "running in the background before you run the tests" because we have our `launch_es`, `launch_weaviate`, etc. scripts that start the containers. The document stores don't need to be running before starting the tests.

Reply:
I see (at least for Elasticsearch) some of the tests aren't using the fixtures running `launch_es`, e.g. `test_elasticsearch_custom_fields` or `test_init_elastic_client`. I am not sure if there's a plan to eventually use the fixture everywhere, but it that's the case, should we keep the requirement of manually starting the containers?

Reply:
Good point. Using the existing fixture in these tests won't work because they need specific settings when the document store i s initialized. However, what would work is to add `launch_es()` as a first line of code in the tests you mentioned. If an elasticsearch instance is already running it will only print a warning but won't do anything else. If an elasticsearch instance is not running yet, it will start one.

Reply:
So I think it's safer to keep the step about running the containers in the docs, as tests using `launch_es` won't be affected and the rest won't fail.

Reply:
Alright. 👍